src/interaction.wiki

```diff
@@ -224,56 +226,64 @@ of user pages on the main page. Here is the new version of the full
 program:
 
 <<code language="ocaml"|
-open Eliom_content.Html5.D
+open Eliom_content.Html.D
+open Eliom_parameter
```

Comment:
Please avoid (globally) opening additional modules, especially `Eliom_parameter` which exports a lot of stuff. In fact, I would go one step further and discourage opening the `Eliom_content` modules.